notebooks/20 Feature Selection.ipynb

```diff
@@ -18,6 +18,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "# SciPy 2016 Scikit-learn Tutorial"
```

Comment:
thanks, I think we didn't cover all of the notebooks, and thus some of them may have had missing titles ... (or we simply forgot)